test/test_puma_server_ssl.rb

```diff
@@ -20,38 +20,46 @@ def ssl_error(server, peeraddr, peercert, error)
 
 class TestPumaServerSSL < Minitest::Test
 
+  # Only @events, @http, and @server need to be instance variables
```

Comment:
👍 but the comment isn't necessary

Reply:
Confused.  Would you like it removed?  I added it for people that may not read the PR/commit description.

Reply:
Just rebased from today's commits and deleted comment...